src/app/layout.tsx

```diff
@@ -14,7 +15,13 @@ export default function RootLayout({ children }: { children: React.ReactNode })
   return (
     <html lang="en" className="light">
       <body className={inter.className}>
-        <Providers>{children}</Providers>
+        <div className="min-h-screen flex flex-col">
+          <Header />
+          <main className="flex-1 overflow-y-auto p-4 max-w-screen-2xl  md:mx-auto">
```

Comment:
Where can I learn more about these classes like 'p-4'?

Reply:
It's just tailwind. Nextui is based on tailwind but for some reason they decided no to provide grid and container components, so I just improvise.